integration_tests/test_suites/backcompat-test-suite/dagit_service/Dockerfile_dagit_source

```diff
@@ -0,0 +1,20 @@
+FROM python:3.8-slim
+
+COPY ./python_modules/ /tmp/python_modules/
+
+WORKDIR /tmp
+
+RUN pip install \
+    -e python_modules/dagster \
+    -e python_modules/dagster-graphql \
+    -e python_modules/dagit \
+    -e python_modules/libraries/dagster-postgres \
+    -e python_modules/libraries/dagster-docker
+
```

Comment:
suggestion: stick a 
```
RUN ! (pip list --exclude-editable | grep -e dagster -e dagit)
```

to make sure we don't accidentally install a dagster dep from pypi here